code/controllers/subsystems/machines.dm

```diff
@@ -47,37 +46,26 @@ SUBSYSTEM_DEF(machines)
 			new_powernet.add_cable(power_cable)
 			propagate_network(power_cable, power_cable.powernet)
 
-/datum/controller/subsystem/machines/stat_entry()
+/datum/controller/subsystem/processing/machines/stat_entry()
 	..("M:[processing.len]|PN:[powernets.len]")
 
-/datum/controller/subsystem/machines/fire(resumed = FALSE)
+/datum/controller/subsystem/processing/machines/fire(resumed = FALSE)
 	if (!resumed)
 		for(var/datum/powernet/powernet as anything in powernets)
 			powernet.reset() //reset the power state.
 		src.currentrun = processing.Copy()
```

Comment:
```suggestion
```

Reply:
copy is needed since processing works off reducing list length gradually.

Reply:
It's not needed here since the parent proc does the same thing.

Reply:
fair point